tenant/service_onboarding_test.go

```diff
@@ -47,3 +48,38 @@ func initOnboardingService(s kv.Store, f influxdbtesting.OnboardingFields, t *te
 
 	return svc
 }
+
+func TestOnboardURM(t *testing.T) {
+	s, _, _ := NewTestInmemStore(t)
+	storage := tenant.NewStore(s)
+	ten := tenant.NewService(storage)
+
+	// we will need an auth service as well
+	svc := tenant.NewOnboardService(ten, kv.NewService(zaptest.NewLogger(t), s))
+
+	ctx := icontext.SetAuthorizer(context.Background(), &influxdb.Authorization{
+		UserID: 123,
+	})
+
+	onboard, err := svc.OnboardUser(ctx, &influxdb.OnboardingRequest{
+		User:   "name",
+		Org:    "name",
+		Bucket: "name",
+	})
+
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	urms, _, err := ten.FindUserResourceMappings(ctx, influxdb.UserResourceMappingFilter{ResourceID: onboard.Org.ID})
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	if len(urms) > 1 {
+		t.Fatal("additional URMs created")
+	}
+	if urms[0].UserID != onboard.User.ID {
+		t.Fatal("org assigned to the wrong user")
+	}
```

Comment:
Take it or leave it. All of this could be reduced to:
```go
urms, _, err := ten.FindUserResourceMappings(ctx, influxdb.UserResourceMappingFilter{ResourceID: onboard.Org.ID})
require.NoError(t, err)
assert.Equal(t, []*influxdb.UserResourceMappings{
    {ResourceID: onboard.Org.ID, UserID: onboard.User.ID},
}, urms)
```